### ISSUE TITLE
[FEATURE] Detecting Noise level to predict maximum possible accuracy of model, add setting to noise suppress

### ISSUE THREAD

**Issue by yovelop** [sdatkinson/neural-amp-modeler#159]
Before training after automatic delay detection try to measure noise of reamp output file by looking in silence in the begin of v1_1_1.wav

1) This level shows error level minimum for the model. We can call it like probable best esr value and show it in start of trainig. For really noisy high-gain reamps this value might be very big like 0.1 ESR or something so user can deside whether to run training on 2000 epochs or stay in current model after 200epochs with the same level of error. Maybe user will try to check his signal chain to fix noise problems...

2) This part of output.wav  where silence in input wav before first pulse signal can be our noise training data to try apply noise reduction methods on output.wav.
So, add setting to apply "noise reduction" to output.wav   with parameter of agrresiveness of noise reduction.
Maybe this will help to train model. 
There are many neural (and not) noise reductors in paid VST but maybe there is open source ones with good accuracy. 

maybe we need longer silence part in the begining of input.wav to get more noise-data

**Comment by sdatkinson** [comment]
Not a bad idea.

One could get the number by:
1. Computing the mean squared amplitude over that interval
2. Computing the m.s.a. over the validation signal.
3. ESRmax=(1) / (2)

Printing this out at the start of training would be good enough, I'd think.

**Comment by sdatkinson** [comment]
Resolved with `v2_0_0.wav` in #217
